packages/std/src/ibc.rs

```diff
@@ -185,6 +187,7 @@ pub struct IbcPacket {
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[non_exhaustive]
```

Comment:
Was any code (outside of cosmwasm-std) creating these manually?

Reply:
I didn't find anything in `wasmvm` or `cosmwasm-plus`. I figured these are only ever deserialized on the Rust side.